packages/drivers/debugger/api-report/debugger.api.md

```diff
@@ -123,10 +123,11 @@ export class DebugReplayController extends ReplayController implements IDebugger
     protected static readonly WindowClosedSeq = -1;
 }
 
-// @internal (undocumented)
+// @alpha (undocumented)
 export namespace FluidDebugger {
```

Comment:
The only two members of this namespace are now marked `@internal`, so does this namespace need to be `@alpha`?

Reply:
Wow!  Nice find.  Loop calls 'createFromServiceFactory(..)'.

I was confused by the namespace (which I thought was a static class) and couldn't figure what I needed to tag to make Loop build.  It was the one place I commented out code to make forward progress.

Not anymore.  :-)